.storybook/preview-head.html

```diff
@@ -0,0 +1,15 @@
+<style>
+  body {
+    /*
+      The <AppDecorator> in preview.tsx should properly set the Inter font as
+      the default font using next/font, but some components are rendered outside
+      of the decorator by being appended to the <body> using a React Portal.
+      Specifically, React Aria does this for modal elements like the <AppPicker>
+      popover. Thus, this is a manual workaround to apply the Inter font there
+      too. On the actual website, outside of Storybook, this should not be a
+      problem, as the font is applied directly to the <body> using next/font in
+      the root layout (src/app/layout.tsx).
+     */
+    font-family: Inter;
+  }
+</style>
```

Comment:
Nit: Missing line at end of file

Reply:
Huh, look like I forgot to enable Prettier for HTML files. Food for a separate PR: 463926139

Reply:
Followup PR: https://github.com/mozilla/blurts-server/pull/3174